eth_account/types.py

```diff
@@ -0,0 +1,31 @@
+from typing import (
+    Dict,
+    Sequence,
+    TypedDict,
+    Union,
+)
+
+from eth_keys.datatypes import (
+    PrivateKey,
+)
+from eth_typing import (
+    HexStr,
+)
+from hexbytes import (
+    HexBytes,
+)
+
+Blobs = Sequence[Union[bytes, HexBytes]]
+Bytes32 = bytes
+PrivateKeyType = Union[Bytes32, int, HexStr, PrivateKey]
+
+
+# from web3.types
```

Comment:
nit: While we're in a beta round, maybe this is worth putting into the current *eth-typing* beta and using it in both *web3.py* and here?